.circleci/config.yml

```diff
@@ -1053,8 +1027,7 @@ workflows:
           filters:
             branches:
               only:
-                - DISABLED_FIXME # master # TODO(AD): See #1174
-          <<: *defaults
+                - master
```

Comment:
WDYT about enabling it on `ad/fix-barretenberg-benchmark-aggregator` as well, so we can test this works before merging, so we accidentally don't break master?

Reply:
Well, auto-merge decided for us :-P